src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  function add(a, b) {
+    return a + b;
+  }
+
+  function subtract(a, b) {
+    return a - b;
+  }
+
+  function multiply(a, b) {
+    return a * b;
+  }
+
+  function divide(a, b) {
+    return a / b;
+  }
+
+  function operate(callback, num) {
+    result = callback(result, num);
```

Comment:
Use the arithmetic assignment operators to simplify the code. For example, `result += num` instead of `result = result + num` inside the `operate` callback functions.